packages/react-native/README.md

```diff
@@ -34,77 +36,21 @@ You can log into your [Honeybadger](https://honeybadger.io) account to obtain yo
 
 
 ## Configuration
+See the [Configuration Reference](https://docs.honeybadger.io/lib/javascript/reference/configuration/) for a full list of config options. 
 
-The configure method takes additional configuration options.
+## Usage
 
-| Name         | Type    | Required | Default | Example              |
-|--------------|---------|----------|---------|----------------------|
-| apiKey       | String  | YES      | `""`    | `"hb-api-key-1234"`  |
-| reportErrors | Boolean | NO       | true    |                      |
-| revision     | String  | NO       | `""`    | `"8afb34a"`          |
-| projectRoot  | String  | NO       | `""`    | `"/path/to/project"` |
+Uncaught iOS, Android, and JavaScript errors will be automatically reported to Honeybadger by default. 
 
-```js
-  Honeybadger.configure(
-    "hb-api-key-1234",
-    "8afb34a",
-    "/path/to/project"
-  );
-```
-
-The **reportErrors** parameter determines if errors are to be sent to Honeybadger. This is set to **true** by default. In certain environments, say, during development, it could be useful to set **reportErrors** to **false** to prevent errors from being posted to your Honeybadger account.
-
-
-## Usage Examples
-
-iOS, Android, and JavaScript errors will be automatically handled by the Honeybadger React Native library, by default. But you can also use the following API to customize error handling in your application.
-
-### Honeybadger.notify(error, additionalData)
```

Comment:
I see why you'd remove this, but I think we should include usage examples of the main features of the package (at least this is what we do with the other packages).
If you look at [React](https://docs.honeybadger.io/lib/javascript/integration/react/) or [Vue](https://docs.honeybadger.io/lib/javascript/integration/vue3/) integration guides, there are a few sections that are always included (Installation, Reporting Errors, Identifying Usage) and then a reference to the js/core package for advanced usage.

Reply:
@BethanyBerkowitz +1 on "... I think we should include usage examples of the main features of the package"

Reply:
Cool, fine by me. It just seemed like copy/pasting the same instructions into all the libraries might be a maintenance headache. But it's definitely easier to read.

Reply:
@subzero10 In the examples you linked, those sections are actually held in the docs repo, not in the READMEs. Happy to put them in the README here, but for consistency I'd put them in `docs` instead.

Reply:
Yes, I posted those to show that even if they inherit most of their features from the js package, they still include sections on the main features.

For the React Native documentation page, the current state is that the documentation is built based on the README. You can change that though.

So, to clarify:
- we can update the React Native README file and make sure the docs page is reflected with the content of the latest README
- we can cleanup the React Native README file (as you did) and make sure that the docs page is on par with the rest of our integration guides

I'm fine with either approach!

Reply:
Here's the documentation PR: https://github.com/honeybadger-io/docs/pull/303